sines.lua

```diff
@@ -538,3 +549,22 @@ function redraw()
   screen.text(math.floor((params:get('output_level')) * 10 / 10) .. " dB")
   screen.update()
 end
+
+function redraw_grid()
+  g:all(0)
+  for x = 1, grid_width do
+    local col_height = grid_height - math.ceil(sliders[x] / grid_slider_scale)
+    for y = col_height, grid_height do
+      g:led(x,y,x == (edit + 1) and 8 or 4)
```

Comment:
PR looks good! Thanks so much. Can you make this small update please. This makes the UI useable for monobright grids. Please verify that it doesn't break varibright grids.

```suggestion
      g:led(x,y,x == (edit + 1) and 8 or grid_bright)
```

and something like this then up top:

```
local g = grid.connect()
local grid_width = g.cols
local grid_height = g.rows
local grid_slider_scale = max_slider_size / grid_height
local grid_bright = 4

-- handle monobright grids
if util.string_starts(g.name, 'monome 64 m64')
  or util.string_starts(g.name, 'monome 128 m128')
  or util.string_starts(g.name, 'monome 256 m256') then
    grid_bright = 15
end
```

these grid names are the older monobright grids.

Reply:
I've slightly changed the logic, but could verify that it now works correctly by manually changing to "monobright" mode. I can't test if monobright grids are detected correctly, but I trust you on that :)